test/Mono.Linker.Tests.Cases/Interop/PInvoke/Warnings/ComPInvokeWarning.cs

```diff
@@ -140,6 +141,17 @@ static void Call_CanSuppressWithRequiresUnreferencedCode ()
 		[DllImport ("Foo")]
 		static extern void CanSuppressWithRequiresUnreferencedCode (IFoo foo);
 
+		[RequiresUnreferencedCode ("test")]
+		static void Call_CanSuppressPInvokeWithRequiresUnreferencedCode ()
+		{
+			CanSuppressPInvokeWithRequiresUnreferencedCode (null);
+		}
+
+		[RequiresUnreferencedCode ("test")]
+		[DllImport ("Foo")]
+		static extern void CanSuppressPInvokeWithRequiresUnreferencedCode (IFoo foo);
+
+
```

Comment:
I would add one more where we expect the warning (not suppress it) just to make sure it works correctly.